Sources/Build/LLBuildCommands.swift

```diff
@@ -259,7 +259,7 @@ final class TestEntryPointCommand: CustomLLBuildCommand, TestBuildCommand {
             #if \#(isXCTMainAvailable)
             \#(generateTestObservationCode(buildParameters: buildParameters))
 
-            import XCTest
+            @_weakLinked import XCTest
```

Comment:
This might be an issue for Windows. There is no such thing as weak linking, everything must be strongly linked.

Reply:
This is just a proof-of-concept branch. Not gonna merge it. :)